3 - Assignment/240714 - Rust Basic + Linux Kernel, Week 2, Day 2/collatz-sequence/src/lib.rs

```diff
@@ -1,6 +1,15 @@
 /// Determine the length of the collatz sequence beginning at `n`.
 pub fn collatz_length(mut n: i32) -> u32 {
-    todo!("Implement this")
+    let mut length = 1_u32;
+    while n != 1 {
```

Comment:
틀린 건 아니지만 다음 코드가 좀 더 명확해 보입니다.
```suggestion
    while n > 1 {
```